src/Umbraco.Abstractions/PublishedCache/IPublishedElementFactory.cs

```diff
@@ -0,0 +1,7 @@
+namespace Umbraco.Web.PublishedCache
+{
+    public interface IPublishedElementFactory
```

Comment:
What is this? this seems to be a brand new addition but is not used anywhere. Perhaps this was the start of trying to resolve this coment? https://github.com/umbraco/Umbraco-CMS/pull/7542#issuecomment-580715415 

In the future, and something we wanted to do in v8 but didn't get a chance - was to overhaul IPublishedContent/IPublishedElement - so that it's just a poco model. Ideally then all traversing and 'doing' operations are done with a service. This would make it so much easier to create a new published content implementation since we would just re-use the poco models and so the models are not also services themselves. But anyways, that's for later. In the meantime, i'm not sure what this is.

Reply:
You are right, it was my first attempt to fix https://github.com/umbraco/Umbraco-CMS/pull/7542#issuecomment-580715415. 
Realised that it was not necessary after I moved PublishedElement to Abstraction. I removed the interface again